docs/azure/migration/appcat/dotnet-cli.md

```diff
@@ -0,0 +1,94 @@
+
+Azure Migrate application and code assessment for .NET is a tool to assess .NET source code to identify replatforming and migration opportunities to Azure. It helps you modernize and replatform large-scale .NET applications through a broad range of transformations, use cases, and code patterns.
+
+This guide describes how to use the application and code assessment Visual Studio extension to both scan and interpret the results of the scan.
+
+If you have not installed the .NET CLI tool, please follow [these instructions first](./install.md#install-the-NET-global-tool).
+
+## Scan your application
+
+The application and code assessment .NET CLI tool lets you decide which projects in your solution to scan to identify migration opportunities to Azure. Follow these steps to scan your application.
+
+1. Switch to the base directory of your solution.
+1. Type `appcat analyze` and press **Enter**.
+
+    > [!NOTE]
+    > If this is your first time running application and code assessment for .NET, you will see an informational message about telemetry and how to opt-out if you should want to.
+
+1. A screen is presented that allows you to pick the projects in your solution to analyze. Use the arrow keys to highlight individual projects and press the spacebar to select them. Press **Enter** when you're ready.
+    ![A screenshot of the CLI tool's project selection](./media/cli/cli-analyze-project-selection.png)
+1. Next you'll be presented with the option to analyze **Source code and settings**, **Binary dependencies**, or both. Make your choice and press **Enter**.
+    ![Screenshot of the CLI dependency selection screen](./media/cli/cli-select-dependencies.png)
+    > [!NOTE]
+    > The **source code and settings** option will only scan the source code in the projects you selected on the previous screen. The **Binary dependencies** option will scan any dependencies (such as NuGet packages) your projects rely on. You can expect to see many more issues identified when **binary dependencies** is selected.
+    > You should consider selecting only **source code and settings** first to figure out what issues reside in your code and fix those before runnig the **binary dependencies** scan.
+    >
+1. You'll then be prompted to generate a report with the results of the analysis. The output can be formatted as CSV, HTML, or JSON. Press **Enter**.
+    ![Screenshot of the CLI report format selection screen](./media/cli/cli-report-format-selection.png)
+1. You'll be prompted for a report name. Input the name and press **Enter**.
+    ![Screenshot of the CLI report name screen](./media/cli/cli-report-name.png)
+1. Finally, you'll be asked whether you want to perform the scan. Press **y** to continue, or **n** to go back and change options.
+1. Once the analysis completes, the report is saved, and a summary of the results are displayed.
+    ![Screenshot of the analysis summary](./media/cli/cli-scan-finished.png)
+
+## Interpret the results
```

Comment:
Both CLI and VS version produce the same reports, so let's move this section as a separate page and call it "Interpret the report results".

We can start with:
"Both CLI tool and Visual Studio allows you to create HTML, CSV and JSON reports. This section describes how to interpret these reports."